idea/testData/intentions/removeArgumentName/mixedNamedArgumentsInTheirOwnPosition.kt

```diff
@@ -0,0 +1,7 @@
+// COMPILER_ARGUMENTS: -XXLanguage:+MixedNamedArgumentsInTheirOwnPosition
+
+fun foo(name1: Int, name2: Int, name3: Int) {}
+
+fun usage() {
```

Comment:
It seems like there are missing test when `MixedNamedArgumentsInTheirOwnPosition` is enabled, but the arguments are not in their own positions, and the intention should be disabled

With your code, this case will look like `foo(1, <caret>name3 = 3, name2 = 2)`